### PR TITLE
Corrosion resistant cargo racks

### PR DESCRIPTION
**Pull request by Luzifer**

### CODE REVIEW

File: modules/internal/cargo_rack.json [hunk 1]
```diff
@@ -8,5 +8,7 @@
     { "id": "05", "edID": 128064343, "eddbID": 1196, "grp": "cr", "class": 6, "rating": "E", "cost": 362590,   "cargo": 64 },
     { "id": "06", "edID": 128064344, "eddbID": 1197, "grp": "cr", "class": 7, "rating": "E", "cost": 1178420,  "cargo": 128 },
     { "id": "07", "edID": 128064345, "eddbID": 1198, "grp": "cr", "class": 8, "rating": "E", "cost": 3829870,  "cargo": 256 }
+    { "id": "08", "edID": 0, "eddbID": 0, "grp": "cr", "class": 1, "rating": "E", "cost": 6250, "cargo": 1, "name": "Corrosion resistant cargo rack" },
```
Comment (Marginal) [inline modules/internal/cargo_rack.json]: Other modules are capitalized, so this should be too - i.e. Corrosion Resistant Cargo Rack

Comment (Luzifer) [inline modules/internal/cargo_rack.json]: fixed

**Comment by Luzifer** [comment]
(Removed index generation from this PR, should be done on master after merging to avoid merge conflicts)

**Comment by Marginal** [comment]
Sorry I wasn't clear -
The "name"s supplied in these files are presented to the user in the website's UI. For consistency with existing modules the names should be capitalised - i.e. the initial letter of each word should be a capital letter.
So these should be "Corrosion Resistant Cargo Rack".

**Comment by Luzifer** [comment]
Hm sure we should include "Cargo Rack" in the naming? In the thrusters file the type is omitted from the naming… (Fixed capitalization)

**Comment by Marginal** [comment]
I agree.

**Comment by Marginal** [comment]
And thanks for this work.

**Pull request merged**